DataFormats/ParticleFlowReco/BuildFile.xml

```diff
@@ -1,7 +1,5 @@
 <use name="DataFormats/CaloRecHit"/>
 <use name="DataFormats/Common"/>
-<use name="DataFormats/EgammaCandidates "/>
```

Comment:
hi @guitargeek - I don't understand this change. DataFormats/EgammaCandidates is used in 3 headers (all of which are then needed to build srcs in this package.

Reply:
Hello! My script is grepping for the exact dependency name, which was here "DataFormats/EgammaCandidates " with a whitespace in the end. By eye this whitespace is not so easy to see when scrolling through the diffs, so I was not prepared for this problem.

I'll make sure to strip the witespaces before grepping next time!

Reply:
ah..I also missed the whitespace.. that explains it:) Thanks.